libkbfs/block_disk_store.go

```diff
@@ -268,6 +276,31 @@ func (s *blockDiskStore) hasData(id BlockID) error {
 	return err
 }
 
+var errFlushedButPresentData = errors.New("Data is flushed but present")
+
+func (s *blockDiskStore) isUnflushed(id BlockID) error {
+	_, err := os.Stat(s.dataPath(id))
+	if err != nil {
+		return err
+	}
+	_, err = os.Stat(s.flushedPath(id))
+	// If the flushed file exists, then the data is not unflushed.
+	if err == nil {
+		return errFlushedButPresentData
+	}
+	return err
```

Comment:
Shouldn't this be `return nil`? Otherwise, if the data exists, we'll always return an error!